stabilize.md

```diff
@@ -43,6 +52,7 @@ Flap Semantics
 
 ```k
     syntax Bid ::= FlapBid ( bid: Wad, lot: Rad, guy: Address, tic: Int, end: Int )
+    syntax Bid ::= FlopBid ( bid: Rad, lot: Wad, guy: Address, tic: Int, end: Int )
```

Comment:
Is it intended that a `FlapBit.lot` is a `Rad`, but the `FlopBid.lot` is a `Wad`?

Reply:
Yes, the lot for a Flap is in Dai, and the lot for a Flop is in MKR.